pkg/apis/core/v1alpha1/types_common.go

```diff
@@ -37,6 +37,9 @@ type LastError struct {
 	// Well-defined error codes of the last error(s).
 	// +optional
 	Codes []ErrorCode `json:"codes,omitempty"`
+	// LastReportTime the time when the error was lastly reported.
+	// +optional
+	LastReportTime metav1.Time `json:"lastReportTime,omitempty"`
```

Comment:
@mvladev do I have to make this variable pointer?

Reply:
Yes

Reply:
Hold on - I think this wraps `time` and it's automatically set to 0

Reply:
Yes, it has a check for `IsZero` https://github.com/kubernetes/apimachinery/blob/d16981aedf33f88b7434bd3bdbcc2e1f940aecd0/pkg/apis/meta/v1/time.go#L61-L66 that is used in (un)marshaling. Then it is should be ok to drop 00391db?

Reply:
I don't have a strong opinion on that, I'm fine if we don't use a pointer, but if you want to introduce it, then it's also okay. Your decision, please adapt the PR accordingly.